hack/update-generated-bindata.sh

```diff
@@ -51,7 +51,8 @@ pushd "${OS_ROOT}" > /dev/null
     examples/sample-app \
     examples/prometheus/... \
     examples/hello-openshift \
-    examples/jenkins/...
+    examples/jenkins/... \
+    examples/templateservicebroker/...
```

Comment:
if this is fundamental to the product, even "examples" doesn't seem like the right location.  Do we have any other templates being used in this way? where do they live?